__tests__/setup.js

```diff
@@ -4,6 +4,11 @@ jest.mock('@react-native-community/async-storage', () => mockAsyncStorage);
 
 jest.mock('react-native-icloudstore', () => mockAsyncStorage);
 
+jest.mock('react-native/Libraries/Utilities/Platform', () => ({
+  OS: 'ios',
+  select: jest.fn(selector => selector.ios),
+}));
+
```

Comment:
So this selects 'ios' in the unit tests? What was selected previously?

Reply:
The default is `ios` to change the Platform such that it runs on `android` so [this](https://github.com/photon-sdk/photon-lib/pull/14/files#diff-a221488ea913610f830199f6b126fe17R8) line uses the GDrive module it needs to be changed to `android`